pkg/hub/client/public/impl_fields.go

```diff
@@ -0,0 +1,100 @@
+package public
+
+import "fmt"
+
+// ifaceRevisionFieldsRegistry holds possible fields configuration for ImplementationRevision query.
+var implRevisionFieldsRegistry = map[ImplementationRevisionQueryFields]string{
+	ImplRevRootFields: `
+		revision`,
+	ImplRevMetadataFields: implRevisionMetadataFields,
+	ImplRevAllFields:      implRevisionAllFields,
+}
+
+var implRevisionMetadataFields = fmt.Sprintf(`
```

Comment:
lines 13-10 were not added but moved from: https://github.com/capactio/capact/pull/443/files#diff-d30b8a24c6a480408c406de4584ea3854c4571d3b6d609b22ba4fb9068d6c73aL32